app/routes/languages.rb

```diff
@@ -5,21 +5,18 @@ module Routes
     class Languages < Core
       get '/languages' do
         tracks = X::Track.all
+        tracks.delete_if { |t| t.repository == 'https://github.com/exercism/xSLUG' }
         active, inactive = tracks.partition { |t| t.active? }
         inactive.sort! { |a, b| b.problems.count <=> a.problems.count }
         erb :"languages/index", locals: { active: active, inactive: inactive }
       end
 
       get '/languages/:track_id' do |track_id|
         track = X::Track.find(track_id)
-        if track.implemented?
-          erb :"languages/language", locals: {
-            track: track,
-            docs: X::Docs::Launch.new(track.repository),
-          }
-        else
-          erb :"languages/not_implemented", locals: { track: track }
```

Comment:
Right, so this patch would need to change some logic so that it would work only with `shakespeare` and not with `teco`.

Do you feel comfortable making that change?